src/home/index.html

```diff
@@ -0,0 +1,56 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="../styles/index.css" />
+    <title>Document</title>
+  </head>
+  <body>
+    <header class="toolbar">
+      <a href="#">My Logo</a>
+      <nav>
+        <img src="../assert/images/toggle.png" alt="Toggle" />
+      </nav>
+    </header>
+    <section class="headline">
+      <p class="heading">I am Mwafrika Glad to see you!</p>

```

Comment:
- [ ] [OPTIONAL] This line is the main title. It should stand out from other texts in the headline section. I suggest you replace the `p` tag with `h1`.

Reply:
I've just changed